src/platform_impl/android/mod.rs

```diff
@@ -359,6 +359,11 @@ impl Window {
         Err(ExternalError::NotSupported(NotSupportedError::new()))
     }
 
+    #[inline]
+    pub fn set_minimized(&self, _minimized: bool) {
+        unimplemented!()
+    }
```

Comment:
Do we want this to panic or should it be a no-op? `set_maximized` below is a no-op.

Reply:
I don't think it particular matters, since the Android backend doesn't even compile right now.